tests/ha/single_pod_restart/test_data_pod_restart.py

```diff
@@ -1699,15 +1745,14 @@ def test_continuous_deletes_during_pod_restart(self):
         assert_utils.assert_equal(wr_bucket - del_bucket, len(buckets),
                                   f"Failed to delete {del_bucket} number of buckets from "
                                   f"{wr_bucket}. Remaining {len(buckets)} number of buckets")
-        LOGGER.info("Step 10: Verified status for In-flight DELETEs while pod was"
+        LOGGER.info("Step 7: Verified status for In-flight DELETEs while pod was"
                     "restarting are successful & remaining buckets count is %s", len(buckets))
-
-        LOGGER.info("Step 11: Verify read on the remaining %s buckets.", buckets)
+        LOGGER.info("Step 8: Verify read on the remaining %s buckets.", buckets)
         rd_output = Queue()
         new_s3data = {}
```

Comment:
make dict()

Reply:
I didn't get any codacy warning for same, so will keep it as is.